lib/js/components/Badges/BadgeScore/BadgeScore.stories.ts

```diff
@@ -0,0 +1,82 @@
+import BadgeScore from './BadgeScore.vue';
+import { BADGE_SCORE_COLORS, BADGE_SCORE_SIZES } from './BadgeScore.consts';
+import { ICONS } from '../../Icon';
+
+import { Meta, StoryFn } from '@storybook/vue';
+
+export default {
+	title: 'Components/Badges/BadgeScore',
+	component: BadgeScore,
+} as Meta<typeof BadgeScore>;
+
+/* INTERACTIVE */
+const StoryTemplate: StoryFn<{
+	text: string;
+	suffix: string;
+	color: string;
+	size: string;
+	icon: string;
+}> = (argTypes) => ({
+	components: { BadgeScore },
+	props: Object.keys(argTypes),
+	template:
+		'<badge-score :color="color" :suffix="suffix" :text="text" :size="size" :icon="ICONS[icon]" />',
+	created() {
+		this.ICONS = ICONS;
+	},
+});
+
+export const Interactive = StoryTemplate.bind({});
+
+Interactive.args = {
+	text: '42',
+	suffix: '%',
+};
+
+Interactive.argTypes = {
+	text: { control: { type: 'text' } },
+	suffix: { control: { type: 'text' } },
+	color: {
+		control: { type: 'select', options: Object.values(BADGE_SCORE_COLORS) },
+		defaultValue: BADGE_SCORE_COLORS.SUCCESS,
+	},
+	icon: {
+		control: { type: 'select', options: [null, ...Object.keys(ICONS)] },
+		defaultValue: 'FA_LIGHTBULB',
+	},
+	size: {
+		control: { type: 'select', options: Object.values(BADGE_SCORE_SIZES) },
+		defaultValue: BADGE_SCORE_SIZES.MEDIUM,
+	},
+};
+
+Interactive.parameters = {
+	design: {
+		type: 'figma',
+		url: 'https://www.figma.com/file/izQdYyiBR1GQgFkaOIfIJI/LMS---DS---Components?node-id=1552%3A34953',
+	},
+};
+
+/* FULL WIDTH */
+const StaticStoryTemplate: StoryFn<{
+	isFullWidth: boolean;
+}> = (argTypes) => ({
+	components: { BadgeScore },
+	props: Object.keys(argTypes),
+	template:
+		'<div v-if="isFullWidth" class="flexWrapper">' +
```

Comment:
@kvas-damian generally we should not use conditions in stories, but in this component we are adding additional wrapper, so this one condition is better than double whole `StoryFn` for only one class.

Reply:
we discussed it off-line, probably there is no need for a wrapper after we change the way we control `width` of this component

Reply:
I think it's the simplest way, minus of this solution is component has unused class:
```
	&.-fullWidth {
		width: 100%;
	}
```
but it's only one rule and we know that this component can be dependent of parent node.

@kvas-damian I still dont know the magic of `min-width` with flex, maybe You mean this example solution:
```
.container .box {
  min-width: 200px;
  max-width: 400px;
  flex-basis: auto; /* default value */
  flex-grow: 1;
}
```
Anyway we don't need it.